index.html

```diff
@@ -3110,6 +3110,12 @@ <h1 class="large-h1">Subscribe to stay informed</h1>
         });
 
         });
-      </script>
+        </script>
+
+        <script>         
```

Comment:
Please remove this opening script tag and merge the two sections together.

Reply:
ok